website-gatsby/src/components/animation-loop-runner.jsx

```diff
@@ -63,7 +63,7 @@ export default class AnimationLoopRunner extends Component {
     // TODO - ideally ocular-gatsby should extract images from example source?
     const {path} = this.props;
     if (path) {
-      const RAW_GITHUB = 'https://raw.githubusercontent.com/uber/loaders.gl/master';
+      const RAW_GITHUB = 'https://raw.githubusercontent.com/uber/luma.gl/master';
```

Comment:
If this logic doesn't work, should we remove it?

Reply:
We could to reduce confusion. We can always add it back if this is fixed in ocular-gatsby.